cartography/intel/gcp/__init__.py

```diff
@@ -55,9 +68,24 @@ def _initialize_resources(credentials):
         crm_v1=_get_crm_resource_v1(credentials),
         crm_v2=_get_crm_resource_v2(credentials),
         compute=_get_compute_resource(credentials),
+        storage=_get_storage_resource(credentials),
     )
 
 
+def _sync_gcp_bucket(neo4j_session, resources, project_id, gcp_update_tag, common_job_parameters):
```

Comment:
I think we can remove this function and replace it by putting 

```
storage.sync_gcp_buckets(neo4j_session, resources.storage, project_id, gcp_update_tag, common_job_parameters)
```
on line 101 and then it should work (after making the linter happy).

This way, `_sync_single_project()` will pass a `project_id` to `storage.sync_gcp_buckets()`.

Reply:
Like this?
```python
def _sync_single_project(neo4j_session, resources, project_id, gcp_update_tag, common_job_parameters):
    compute.sync(neo4j_session, resources.compute, project_id, gcp_update_tag, common_job_parameters)
    storage.sync(neo4j_session, resources.storage, project_id, gcp_update_tag, common_job_parameters)
````

What if someone just wants to `sync` projects? Wouldn't they inadvertently also `sync` buckets because it was abstracted away? 

If the reason for this is to ensure that `projects` are up-to-date before syncing `buckets`, maybe we could add the following to `_sync_gcp_bucket()`: 

```python
def _sync_gcp_bucket(neo4j_session, resources, project_id, gcp_update_tag, common_job_parameters):
    _sync_single_project(neo4j_session, resources, project_id, gcp_update_tag, common_job_parameters)
    storage.sync(neo4j_session, resources.storage, project_id, gcp_update_tag, common_job_parameters)
````
Maybe I'm not thinking about this right, but I'd appreciate your insight